default.nix

```
@@ -39,6 +39,8 @@ let
           doCheck = false;
         });
         hnix-store-core = self.callHackage "hnix-store-core" "0.1.0.0" {};
+
+        ghcid = haskellLib.justStaticExecutables (self.callCabal2nix "ghcid" (hackGet ./dep/ghcid) {});
```

Better to do this at use site.
```suggestion
        ghcid = self.callCabal2nix "ghcid" (hackGet ./dep/ghcid) {};
```